completions/carton

```diff
@@ -72,6 +72,11 @@ _comp_cmd_carton()
     fi
 
     case $command in
+        exec)
+            COMPREPLY=($(for i in $(compgen -f -d -- ${cur}); do if test -d "$i"; then echo "$i/"; elif test -x "$i"; then echo "$i "; fi; done));
+            test -d "${COMPREPLY[0]}" && compopt -o nospace
```

Comment:
Thank you for your contribution. I guess the behavior you expect could be achieved by just the following. Could you check it?

```suggestion
            _comp_compgen_filedir
```

Reply:
Success!

Reply:
Thanks for updating.